src-ui/lib/db/schema.ts

```diff
@@ -14,4 +14,5 @@ export const books = sqliteTable('books', {
   copyrightDate: text(),
   coverImages: text({ mode: 'json' }).$type<{ small?: string; medium?: string; large?: string }>(),
   hasRead: integer({ mode: 'boolean' }),
+  scanDate: text(),
```

Comment:
🤔 since not all books will _have_ `isbn` numbers (anything older than the 70s won't); and we will definitely have some books manually added rather than scanned -- should this be named `createdAt` instead of `scanDate`?

We can do the Rails pattern of all DB tables having `createdAt` (and perhaps `updatedAt`) columns.